integration-tests/emulated/src/tests/peregrine/did.rs

```diff
@@ -56,19 +45,36 @@ fn test_did_creation_from_asset_hub() {
 			require_weight_at_most,
 			call,
 		},
-	]));
+	]))
+}
+
+fn get_destination() -> VersionedMultiLocation {
```

Comment:
Rather `get_peregrine_as_sibling_location`, I'd say.

Reply:
It is updated, but I can not find the commit sorry :smile: